tests/integration/test_interaction.py

```diff
@@ -89,6 +90,17 @@ def _get_zone_position(self, zone_id):
             'return $("div[data-uid=\'{zone_id}\']").prevAll(".zone").length'.format(zone_id=zone_id)
         )
 
+    @staticmethod
+    def wait_until_ondrop_xhr_finished(elem):
+        """
+        Waits until the XHR request triggered by dropping the item finishes loading.
+        """
+        wait = WebDriverWait(elem, 2)
+        # While the XHR is in progress, a spinner icon is shown inside the item.
+        # When the spinner disappears, we can assume that the XHR request has finished.
+        wait.until(lambda e: 'fa-spinner' not in e.get_attribute('innerHTML'),
+                   u"Spinner should not be in {}".format(elem.get_attribute('innerHTML')))
```

Comment:
@itsjeyd No problem, I should have noticed it when implementing the function :) Fixed.